src/test/interface/io/json/json_parser_test.cpp

```diff
@@ -215,7 +215,7 @@ TEST(ioJson,jsonParserO4) {
              << 4.01001 
              << "E:arr" 
              << "E:obj" << "E:text";
-  test_parser("{ \"foo\" : 1, \n \"baz\" : [ 2, 3, 4.01001 ] };",
+  test_parser("{ \"foo\" : 1, \n \"baz\" : [ 2, 3, 4.01001 ] }",
```

Comment:
Nothing is allowed after the top level comment, so I had to delete ";" fromt the test.

Reply:
I was a Java/C++ programmer when I wrote those tests - couldn't imagine not putting a semi-colon at the end of something.